src/codegen/tests/debug_tests.rs

```diff
@@ -342,3 +342,142 @@ fn switch_case_debug_info() {
     !9 = !DILocation(line: 0, scope: !4)
     "###);
 }
+
+#[test]
+fn dbg_declare_has_valid_metadata_references_for_methods() {
+    let codegen = codegen_with_debug_version(
+        r"
+        FUNCTION_BLOCK fb
+        METHOD foo
+        END_METHOD
+        END_FUNCTION_BLOCK
+        ",
+        4,
```

Comment:
is there any particular reason to override the debug version here?

Reply:
None, I just copy-pasted the previous test and modified it. Good catch though, that should be probably just be a `codegen` call